checker/examples/subtyping-extension/qual/Encrypted.java

```diff
@@ -1,15 +1,16 @@
 package qual;
 
 import org.checkerframework.framework.qual.*;
+import org.checkerframework.framework.qual.LiteralKind;
```

Comment:
There is both an import for `qual.*` and `qual.LiteralKind`. Can we make all imports explicit?